JPF/mysql.skeleton.ipynb

```diff
@@ -108,6 +107,11 @@
     "pw = os.environ['MYSQL_PASSWORD']\n",
     "db = os.environ['MYSQL_DATABASE']\n",
     "\n",
+    "user = 'root'\n",
+    "host = 'localhost'\n",
+    "pw = ''\n",
+    "db = 'heatseek'\n",
+    "\n",
```

Comment:
i did but I need to restart my ipython process to re-read the env when i switch between AWS/Local .. easier this way where AWS is in hidden ENV vars and I can easily overwrite with locals .. i know this is probably bad form, I didn't intend for them to get committed.